pkgs/by-name/ee/ee/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  ncurses,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ee";
+  version = "1.5.2";
+
+  src = fetchgit {
+    url = "https://git.freebsd.org/src.git";
+    tag = "release/14.3.0";
+    outputHash = "sha256-nMhHXeoam9VtUuhhi0eoGZfcW9zZhpYQKVYbkAbfgc0=";
+    rootDir = "contrib/ee";
+  };
+
+  buildInputs = [ ncurses ];
+
+  postPatch = ''
+    substituteInPlace create.make --replace-fail "/usr/include/curses.h" "${ncurses.dev}/include/ncurses.h"
+    substituteInPlace create.make --replace-fail "-lcurses" "-lncurses"
+  '';
+
+  NIX_CFLAGS_COMPILE = "-DHAS_UNISTD=1 -DHAS_STDLIB=1 -DHAS_SYS_WAIT=1";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ee $out/bin/ee
+  '';
+
+  meta = {
+    description = "ee – a classic curses text editor";
+    homepage = "https://man.freebsd.org/cgi/man.cgi?ee%281%29";
```

this seems to work just as fine
```suggestion
    homepage = "https://man.freebsd.org/cgi/man.cgi?ee";
```